### DISCUSSION TITLE
Apply Rule - Default dates range

### DISCUSSION THREAD

**Discussion by top-cg** [firefly-iii/firefly-iii#6089] [Firefly III]
Hello,

I searched the documentation for setting the default dates range when running a rule manually and not found what i was searching.

Do you know if it is possible to have the range matching the current selected period (and not the full database date range) ? 

![image](https://user-images.githubusercontent.com/18399003/169691087-eef11fa8-7f56-4566-bcce-ba6a6da10c90.png)

**Comment by JC5** [comment]
I'm afraid not, but I have improvements set for the new release!
